src/components/Header/Header.jsx

```diff
@@ -1,12 +1,11 @@
-import "../Header/header.css";
+import "../Header/Header.css";
 import Nav from "../Nav/Nav";
 import SearchForm from "../SearchForm/SearchForm";
```

Comment:
If you have `onChange` or ` onInput` in an input then you need to add the `value` attribute to this input.
This is how React controls inputs.

To read more about **controlled components**: https://reactjs.org/docs/forms.html#controlled-components


Please, add it into the input of `SearchForm`